Content.Shared/Mobs/Components/MobStateComponent.cs

```diff
@@ -13,30 +13,22 @@ namespace Content.Shared.Mobs.Components
     /// </summary>
     [RegisterComponent]
     [NetworkedComponent]
+    [AutoGenerateComponentState(true)]
```

Comment:
```suggestion
    [NetworkedComponent]
    [AutoGenerateComponentState]
```

You're not using this handler.